### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pitdicker**

### CODE REVIEW

File: parse-zoneinfo/README.md [hunk 1]
```diff
@@ -1,42 +1,35 @@
 # parse-zoneinfo
 
-Parse-zoneinfo is a fork of [`zoneinfo_parse`][zoneinfo_parse], with adjustments such that it no longer depends on the `Datetime` crate. It is used by [`chrono-tz`][chrono_tz].
+Rust library for reading the text files comprising the [zoneinfo database], which records time zone changes and offsets across the world from multiple sources.
```
Comment (djc) [inline parse-zoneinfo/README.md:3]: Can you turn this into smaller commits that do one thing? I don't know what I'm reviewing for here.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
 # Chrono-TZ
 
-`Chrono-TZ` is a library that provides implementors of the
-[`TimeZone`][timezone] trait for [`rust-chrono`][chrono]. The
-impls are generated by a build script using the [`IANA database`][iana]
-and [`parse-zoneinfo`][parse_zoneinfo].
+`Chrono-TZ` is a library that provides implementors of the [`TimeZone`] trait for [chrono]. The
```
Comment (djc) [inline README.md:3]: Please don't reflow if the commit message is "Fix dead links"...

File: README.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ build:
 
 ```toml
 [dependencies]
-chrono-tz = { version = "0.6", features = [ "filter-by-regex" ] }
+chrono-tz = { version = "0.9", features = [ "filter-by-regex" ] }
```
Comment (djc) [inline README.md:155]: Let's just ditch this bit of README, I think everyone knows how to do this now.

File: parse-zoneinfo/README.md (hunk 1 above)
Comment (pitdicker) [reply parse-zoneinfo/README.md:3]: Split into three commits.

File: README.md (hunk 2 above)
Comment (pitdicker) [reply README.md:3]: It didn't feel significant enough for a separate commit... I adjusted the commit message. Is that okay?

File: README.md (hunk 3 above)
Comment (pitdicker) [reply README.md]: Done.

File: parse-zoneinfo/README.md (hunk 1 above)
Comment (djc) [reply parse-zoneinfo/README.md:3]: Sorry, but this first commit still looks like a grab bag of unrelated improvements to me. Want to split it some more?

Comment (pitdicker) [reply parse-zoneinfo/README.md:3]: Done :disappointed:.

**Review by djc — APPROVED** [review]

File: parse-zoneinfo/README.md [hunk 4]
```diff
@@ -14,8 +14,6 @@ Parse-zoneinfo is a fork of [`zoneinfo_parse`] by Benjamin Sago (now unmaintaine
 [`zoneinfo_parse`]: https://crates.io/crates/zoneinfo_parse
 [`chrono-tz`]: https://crates.io/crates/chrono-tz
 
-### [View the Rustdoc](https://docs.rs/parse-zoneinfo)
```
[the remark this answers is not in the document]
Comment (pitdicker) [reply parse-zoneinfo/README.md:16]: That is something I also wanted to see and forgot about! I'll open a fresh PR for that.

**Pull request merged**